model-desc/README.md

```diff
@@ -52,3 +52,25 @@ In support of data submission, data loading templates can be downloaded, either
+
+- **_program\_acronym_** is the key property for the node **_program_**. It is the name of the program under which related studies will be grouped, expressed in the form of the acronym by which it will identified within the UI. This property is used as the key via which study records can be associated with the appropriate program during data loading, and to identify the correct records during data updates.
+- **_phs\_accession_** is the key property for the node **_study_**. It is the PHS accession number (a.k.a dbGaP accession).
+- **_study\_participant\_id_** is the key property of the node **_participant_**. The property **_study\_participant\_id_** is a compound property, combining the property **_participant\_id_** and the parent property **_study.phs\_accession_**.
+- **_study\_diagnosis\_id_** is the key property for the node **_diagnosis_**. The property **_study\_diagnosis\_id_** is a compound property, combining the property **_diagnosis\_id_** and the parent property **_participant.study\_participant\_id_**.
+- **_sample\_id_** is the key property of the node **_sample_**. It is the sample identifier as submitted by requestor.
+- **_treatment\_id_** is the key property of the node **_treatment_**. The property **_treatment\_id_** is a compound property, combining the parent property **_participant.study\_participant\_id_**, string character "\_" in the middle as the connector, and the property **_therapeutic\_agents_**.
+- **_file\_id_** is the key property of the node **_file_**. It is the file identifier.
+- **_genomic\_info\_id_** is the key property of the node **_genomic\_info_**. It is the genomic info identifier. If the value is missing, but the **_library\_id_** values are given, will replace the **_genomic\_info\_id_** value with the combination of the parent node id value **_file\_id_** and **_library\_id_**; if the **_library\_id_** is also missing, will replace the value with the parent id value **_file\_id_**.
+- **_study\_link\_id_** is the key property of the node **_image_**. It should consist of a string and a number.
+- **_MultiplexMicroscopy\_id_** is the key property of the node **_MultiplexMicroscopy_**. It is the MultiplexMicroscopy identifier.
+- **_NonDICOMCTimages\_id_** is the key property of the node **_NonDICOMCTimages_**. It is the NonDICOMCTimages identifier.
+- **_NonDICOMpathologyImages\_id_** is the key property of the node **_NonDICOMpathologyImages_**. It is the NonDICOMpathologyImages identifier.
+- **_NonDICOMPETimages\_id_** is the key property for the node **_NonDICOMPETimages_**. It is the NonDICOMPETimages identifier.
+- **_NonDICOMradiologyAllModalities\_id_** is the key property of the node **_NonDICOMradiologyAllModalities_**. It is the NonDICOMradiologyAllModalities identifier.
+- **_NonDICOMMRimages\_id_** is the key property for the ndoe **_NonDICOMMRimages_**. It is the NonDICOMMRimages identifier.
```

Comment:
Line 74 "ndoe" beeds to be updated to "node"

Reply:
It is fixed